### DISCUSSION TITLE
购买了一年专业版，目前不能使用，麻烦发送一下激活码到邮箱，谢谢

### DISCUSSION THREAD

**Discussion by zbstao** [getlantern/lantern-forum-cn#609] [Q&A]
微信支付，邮箱 zhongbin757@hotmail.com
![微信图片_20240619093613](https://github.com/getlantern/lantern-forum-cn/assets/6638997/2aea0bde-d5bd-43c6-b6b1-d3a78f10fe1d)

**Comment by MaxMKMax** [comment]
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
@zbstao 您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.
